odl/discr/discretization.py

```diff
@@ -214,6 +212,11 @@ def __eq__(self, other):
                 other.interpolation == self.interpolation)
 
     @property
+    def impl(self):
+        """The underlying implmentation type for the dspace."""
```

Comment:
Drop "The", and replace "dspace" by something more telling, e.g. "data storage"?